### ISSUE TITLE
Add Insurance address information to demographics report

### ISSUE THREAD

**Issue by teryhill** [LibreHealthIO/LibreEHR#778]
User requested the Insurance address information on the report

Before

![image](https://user-images.githubusercontent.com/4655366/33464758-08933f9e-d612-11e7-94e6-609707cfb635.png)

After

![image](https://user-images.githubusercontent.com/4655366/33464712-bf29800c-d611-11e7-9395-05311ddb239a.png)

**Comment by Jdew192837** [comment]
I plan to work on this as part of GCI

**Comment by teryhill** [comment]
@Jdew192837 this is already done it is just waiting on the core group to vote on whether it is a good thing to add.

**Comment by htuck** [comment]
I don't know if I get a vote but it seems like a good idea to me.  Whoever this report goes to probably would want the real ins co information instead of just what the clinic calls it internally.

**Comment by aethelwulffe** [comment]
Dude.  Add it.
I have added this as custom code in the past.

**Comment by aethelwulffe** [comment]
So long as it has a globals toggle, it is great.

**Comment by teryhill** [comment]
PR Merged Issue closed
